misc/scripts/install-local.sh

```diff
@@ -154,6 +154,22 @@ function compare_remote_version() (
     fi
 )
 
+function get_incompatible_releases() {
+    local distro_name="$(lsb_release -si 2> /dev/null | tr '[:upper:]' '[:lower:]')"
+    if [[ "$(lsb_release -ds 2> /dev/null | tail -c 4)" == "sid" ]]; then
+        local distro_version_name="sid"
+    else
+        local distro_version_name="$(lsb_release -sc 2> /dev/null)"
+    fi
+    local distro_version_number="$(lsb_release -sr 2> /dev/null)"
+    local input=("$@")
+    for key in "${input[@]}"; do
+        if [[ $key == "${distro_name}:${distro_version_name}" ]] || [[ $key == "${distro_name}:${distro_version_number}" ]]; then
```

Comment:
`key` should be normalized to lowercase, that way both `Ubuntu:Jammy` and `ubuntu:jammy` would work as key